.github/workflows/nodejs.yml

```
@@ -55,3 +55,30 @@ jobs:
           install-command: npm ci --foreground-scripts
       - name: Lint
         run: npm run lint
+
+  validate-lockfile:
+    runs-on: ubuntu-latest
+    env:
+      FORCE_COLOR: 1
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Upgrade npm # for workspace support
+        run: npm i -g npm
```

```suggestion
        run: npm i -g npm@${{ matrix.npm-version }}
```

Shouldn't this in any case be pinned to a known major version like in rest of CI (default `npm@9`)? Or explicitly `npm@latest` to make that intention clear.

---

I'm fine with installing whatever npm comes with whatever version we use for this